tools/archives/pyvo_integration/tool-data/astronomical_archives_gen.loc

```diff
@@ -0,0 +1,139 @@
+# Table used for listing astronomical archives TAP service urls
+#<id>	<display_name>	<value>\n0	AIP DaCHS TAP service	http://gavo.aip.de/tap
```

Comment:
this `\n` before line 0 seems suspicious

Reply:
yea looks like a space has been skipped during generation of the file, it doesn't seem to cause bugs except for the fact that this archive one is not available on the list. Fix has been pushed.

But I don't think it's related to the failed tests, when I run the thing locally with planemo test or serve everything works fine. The data table is implemented a bit differently whether  the tool is directly inside galaxy or from the tool shed but both worked when I tested it and the tests failed the same way before any changes.

The problem seems to be docker related, exit code 125 so I don't know if it's a tool problem or a problem with the workflow. 

I tried to look at planemo source and search if I found other people with same problem but have found nothing so far.

I looked at the logs from your cta and there's no error message after the container is instantiated

Reply:
For the error message, it's a separate point indeed, let's ask developers after this is ready.

Reply:
That's the only problem remaining, everything else seems to pass now. Maybe the container failling is not a problem in itself but the fact that it writes to the error stream makes galaxy think tool run has failed, will try to change how the tool react to something written to error stream.

Did you update the workflow between now and your latest cta tool test?